sdk/python/lib/pulumi/automation/_server.py

```diff
@@ -98,6 +99,7 @@ def Run(self, request, context):
             loop.close()
             sys.stdout.flush()
             sys.stderr.flush()
+            RPC_MANAGER.clear()
```

Comment:
Should the doc comment on `clear` be updated now that this is not just used for tests?